lib/find_specific_terms.rb

```diff
@@ -0,0 +1,67 @@
+require 'csv'
+
+class FindSpecificTerm
+  CSV_HEADERS = ["Title", "URL", "Publishing application", "Tagged organisation", "Format", "Content ID"].freeze
+
+  def call
+    write_csv
+  end
+
+  def self.call(*args)
+    new(*args).call
+  end
+
+  private_class_method :new
+
+private
+
+  def write_csv
+    CSV.open("#{Rails.root}/tmp/search_term_content_items.csv", 'wb') do |csv|
+      csv << CSV_HEADERS
+
+      searchterm = %(another EU country)
```

Comment:
If you want a single string search term, you could use single or double quotes instead. This also means you don't need the `searchterm.each do |term|` block below - the contents of the block can just use `searchterm` directly.

Reply:
Also, `searchterm` could be provided each time this class is instantiated so that you can provide your own search term when you call this class via a rake task rather than having to change it in the code.